docs/OffsetHelper.md

```diff
@@ -333,6 +333,562 @@ Retire the specified TCO2 tokens.
+Use `calculateNeededETHAmount()` first in order to find out how much
+MATIC is required to retire the specified quantity of TCO2.
+
+This function:
+1. Swaps the Matic sent to the contract for the specified pool token.
+2. Redeems the pool token for the poorest quality TCO2 tokens available.
+3. Retires the TCO2 tokens.
+
+_If the user sends much MATIC, the leftover amount will be sent back
+to the user._
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| _poolToken | address | The address of the Toucan pool token that the user wants to use, for example, NCT or BCT. |
+| _amountToOffset | uint256 | The amount of TCO2 to offset. |
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| tco2s | address[] | An array of the TCO2 addresses that were redeemed |
+| amounts | uint256[] | An array of the amounts of each TCO2 that were redeemed |
+
+### autoOffsetExactInETH
+
+```solidity
+function autoOffsetExactInETH(address _poolToken) public payable returns (address[] tco2s, uint256[] amounts)
+```
+
+Retire carbon credits using the lowest quality (oldest) TCO2
+tokens available from the specified Toucan token pool by sending MATIC.
+All provided MATIC is consumed for offsetting.
+
+This function:
+1. Swaps the Matic sent to the contract for the specified pool token.
+2. Redeems the pool token for the poorest quality TCO2 tokens available.
+3. Retires the TCO2 tokens.
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| _poolToken | address | The address of the Toucan pool token that the user wants to use, for example, NCT or BCT. |
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| tco2s | address[] | An array of the TCO2 addresses that were redeemed |
+| amounts | uint256[] | An array of the amounts of each TCO2 that were redeemed |
+
+### autoOffsetPoolToken
+
+```solidity
+function autoOffsetPoolToken(address _poolToken, uint256 _amountToOffset) public returns (address[] tco2s, uint256[] amounts)
+```
+
+Retire carbon credits using the lowest quality (oldest) TCO2
+tokens available by sending Toucan pool tokens, for example, BCT or NCT.
+
+This function:
+1. Redeems the pool token for the poorest quality TCO2 tokens available.
+2. Retires the TCO2 tokens.
+
+Note: The client must approve the pool token that is sent.
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| _poolToken | address | The address of the Toucan pool token that the user wants to use, for example, NCT or BCT. |
+| _amountToOffset | uint256 | The amount of TCO2 to offset. |
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| tco2s | address[] | An array of the TCO2 addresses that were redeemed |
+| amounts | uint256[] | An array of the amounts of each TCO2 that were redeemed |
+
+### isEligible
+
+```solidity
+function isEligible(address _erc20Address) private view returns (bool)
+```
+
+Checks whether an address can be used by the contract.
```

Comment:
Can be used by the contract for what? Would be helpful to clarify the context rather than assume knowledge.